.github/workflows/track-acvm.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         run: |
           # Extract acvm version from the current Cargo.lock
           CURRENT_VERSION=$(awk '/name = "acvm"/ {getline; print $3}' Cargo.lock | tr -d '"')
-          INTEGRATION_TEST_VERSION=$(yarn workspace integration-tests info @noir-lang/acvm_js --json | jq .children.Version | tr -d '"')
+          INTEGRATION_TEST_VERSION=$(awk '/@noir-lang\/noir_js@workspace:/ {flag=1; next} /noirc_abi: "workspace:*/ {flag=0} flag' yarn.lock | grep -Eo '"@noir-lang/acvm_js": [0-9.]+[^,]' | awk -F': ' '{print $2}' | tr -d ' "')
```

Comment:
What's the benefit of this?

Reply:
The integration test no longer has a version of acvm_js to pick up the version number from. I now get it from yarn.lock. I could however get it from noir_js package.json. Thoughts?

Reply:
Yeah, pulling it from `noir_js` would be my preference on this. All this `awk`ing isn't particularly readable.

Reply:
Please recheck